provisioning/requirements.yml

```diff
@@ -0,0 +1,8 @@
+- src: https://github.com/idi-ops/ansible-facts
```

Comment:
Ok, according to http://docs.ansible.com/ansible/YAMLSyntax.html is optional, so nevermind if you made it on purpose.